microprofile/fault-tolerance/src/main/java/io/helidon/microprofile/faulttolerance/FaultToleranceExtension.java

```diff
@@ -260,33 +332,31 @@ public Type getBaseType() {
             return delegate.getFields();
         }
 
-        /**
-         * Gets the annotation.
-         *
-         * @param annotationType Annotation type.
-         * @param <R> Annotation type param.
-         * @return Annotation instance.
-         */
+        @Override
         @SuppressWarnings("unchecked")
         public <R extends Annotation> R getAnnotation(Class<R> annotationType) {
-            if (CommandBinding.class.equals(annotationType)) {
-                return (R) LiteralCommandBinding.getInstance();
-            }
-            return delegate.getAnnotation(annotationType);
+            Optional<Annotation> optional = annotationSet.stream()
+                    .filter(a -> annotationType.isAssignableFrom(a.annotationType())).findFirst();
```

Comment:
Will the formatting police want the `.findFirst` to be on the next line by itself??

Reply:
I'm not sure, but I'll move it.